constants/Theme.ts

```diff
@@ -15,6 +15,17 @@ export const PrimaryColorStyle = StyleSheet.create({
   },
   border: {
     borderColor: PrimaryColor
+  },
+  button: {
+    backgroundColor: PrimaryColor,
+    color: '#ffffff'
+  }
```

Comment:
don't use feature style, should only declare utility style

Reply:
aka more into component level

Reply:
ok copy